### PR TITLE
Tweak probe values and add FailureThreshold

### PR DESCRIPTION
**Pull request by kerneltime**
Make probes more eager.

Signed-off-by: Ritesh H Shukla <ritesh@minio.io>

### CODE REVIEW

File: examples/minioinstance.yaml [hunk 1]
```diff
@@ -114,18 +114,20 @@ spec:
   ## is not working properly and needs restart. Kubernetes automatically
   ## restarts the pods if liveness checks fail.
   liveness:
-    initialDelaySeconds: 120
-    periodSeconds: 15
-    timeoutSeconds: 12 # should always greater than MINIO_API_READY_DEADLINE (which defaults to 10s)
+    initialDelaySeconds: 35
+    periodSeconds: 5
+    failureThreshold: 1000
+    timeoutSeconds: 60 # should always greater than MINIO_API_READY_DEADLINE (which defaults to 10s)
```
Comment (harshavardhana) [inline examples/minioinstance.yaml:120]: This comment is for readiness only not liveness @kerneltime :-) it was a mistake from previous PR

File: minio-operator.yaml [hunk 2]
```diff
@@ -196,7 +196,7 @@ spec:
       containers:
         - name: minio-operator
           image: minio/k8s-operator:2.0.6
-          imagePullPolicy: IfNotPresent
+          imagePullPolicy: Always
```
Comment (harshavardhana) [inline minio-operator.yaml:199]: shouldn't this be? IfNotPresent?

Comment (kerneltime) [reply minio-operator.yaml:199]: I don't have an opinion other than it helps with debugging locally.

File: examples/minioinstance.yaml (hunk 1 above)
Comment (kerneltime) [reply examples/minioinstance.yaml:120]: Will fix it :-)

**Comment by kerneltime** [comment]
#159 supersedes this

**Pull request closed without merging**